docs/ref/contrib/admin/index.txt

```diff
@@ -2829,6 +2835,45 @@ The presence of the ``admin_password_reset`` named URL will cause a "forgotten
 your password?" link to appear on the default admin log-in page under the
 password box.
 
+.. _overriding-default-adminsite:
+
+Overriding default ``AdminSite``
+--------------------------------
+
+.. versionadded:: 1.11
+
+You can override the default `django.contrib.admin.sites.AdminSite` instance with your own:
+just write your own `AdminConfig` and set the `SimpleAdminConfig.default_site`
+with a dotted path string to your custom `AdminSite`.
+
+.. snippet::
+    :filename: myproject/admin.py
+
+    class MyAdminSite(admin.AdminSite):
+        ...
+
+.. snippet::
+    :filename: myproject/apps.py
+
+    from django.contrib.admin.apps import AdminConfig
+
+    class MyAdminConfig(AdminConfig):
+        default_site = 'myproject.admin.MyAdminSite'
+
+.. snippet::
+    :filename: myproject/settings.py
+
+    INSTALLED_APPS = [
+        ...
+        #'django.contrib.admin.apps.AdminConfig',
+        'myproject.apps.MyAdminConfig',
+        ...
+    ]
+
+.. note::
+
+    Default admin site search for an AppConfig with label == 'admin'
```

Comment:
I don't quite understand this comment - where does the `AdminSite` search for the appconfig with this label?